pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -64,6 +64,8 @@
 
 public class MultiTopicsConsumerImpl<T> extends ConsumerBase<T> {
 
+    public static final String FAKE_TOPIC_NAME_PREFIX = "TopicsConsumerFakeTopicName";
```

Comment:
I'd actually get rid of "FakeTopicName" alltogether. It is super-confusing to look at in the logs.. and it doesn't make any sense in the context.

Reply:
I think this should be completely removed if it doesn't any make sense. However, if a list of multiple topics is specified, what string should we set as a single topic name?

Reply:
Maybe return `"MultiTopicConsumer"`? That would at least hint at the reason why this consumer doesn't have a single topic. When getting `FakeTopicName`, I'd think someone has indeed created that topic.

Reply:
Changed the prefix of dummy topic name to `MultiTopicsConsumer-`. What do you think?

Reply:
Thanks. I saw the "fake topic" thing a few weeks back in logs and got very confused (then I forgot to actually fix it).